packit/api.py

```diff
@@ -2557,7 +2558,11 @@ def clean(self):
             self.up.local_project.free_resources()
 
     @staticmethod
-    def validate_package_config(working_dir: Path, offline: bool = False) -> str:
+    def validate_package_config(
+        working_dir: Path,
+        offline: bool = False,
+        config: Optional[str] = None,
+    ) -> Optional[str]:
```

Comment:
I would maybe suggest using `config_path` or something like that to explicitly show it is a file path that we are getting here...

Also, can you please add a new entry to the docstring bellow?